nativelib/src/main/scala/scala/scalanative/posix/pthread.scala

```diff
@@ -72,6 +75,9 @@ object pthread {
                             stackaddr: Ptr[Byte],
                             stacksize: CSize): CInt = extern
 
+  def pthread_attr_setstackaddr(attr: Ptr[pthread_attr_t],
+                                stackaddr: Ptr[Byte]): CInt = extern
+
   def pthread_attr_setstacksize(attr: Ptr[pthread_attr_t],
                                 stacksize: CSize): CInt = extern
 
```

Comment:
I don't see why not. `java.util.Thread` was implemented in the 90's based on functionality equivalent to that available in SUSv2. Should be just fine.